src/components/CustomTextField.jsx

```diff
@@ -0,0 +1,38 @@
+import { FormControl, Typography, InputBase, FormHelperText } from "@mui/material";
+import PropTypes from "prop-types";
+
+const CustomTextField = ({ label, helperText, handleChange, ...rest }) => {
+  return (
+    <FormControl fullWidth>
+      <div style={{ display: "flex", alignItems: "center", flexWrap: "nowrap" }}>
+        <Typography style={{ marginRight: "8px", whiteSpace: "nowrap", minWidth: "100px" }}>{label}</Typography>
+        <InputBase
+          {...rest}
```

Comment:
I think its better to use styled function for all the custom fields and also 
```javascript
{...rest}
``` 
it should be the last in the component